internal/cluster/cluster.go

```diff
@@ -541,6 +541,14 @@ func CanDownloadKubeconfig(c *common.Cluster) (err error) {
 }
 
 func (m *Manager) IsOperatorAvailable(c *common.Cluster, operatorName string) bool {
+	// TODO: MGMT-4458
+	// Backward-compatible solution for clusters that don't have monitored operators data
+	if len(c.MonitoredOperators) == 0 {
```

Comment:
Shouldn't we check for that specific operator according to `operatorName` parameter?
Are we assuming, that each cluster will have all operators or non at all?!

Reply:
This function is only used by the Console... But yes. Normally - There must be at least 1 built-in operator (console) if not 2 (CVO when CHECK_CLUSTER_VERSION is turned on) being monitored.
The built-in operators are defaults, while OLM are customized.